app/(default)/page.tsx

```diff
@@ -1,11 +1,15 @@
+import Image from 'next/image';
 import Link from 'next/link';
 import { ReactElement } from 'react';
 
+import logo from '../assets/nilshindermann_logo.svg';
+
 export default function Home(): ReactElement {
     return (
         <>
-            <h1 className="font-sans text-4xl font-bold text-primary md:text-6xl">
-                Nils Hindermann
+            <Image className="mb-8" src={logo} alt="nils hindermann" />
```

Comment:
Image looks great, but it seems a bit weird having your name in the title and then in the introduction.

Reply:
This is what I mean: 
![image](https://github.com/nilshindermann/nilsh.ch/assets/76178668/8d18a125-22b6-4fdb-ab53-21e16fbb2be0)